### ISSUE TITLE
Capture research filters in URL

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-app#823]
Currently the URL only captures the search query, e.g. https://policyengine.org/us/research?search=child+tax+credit

Filtering by topic, location, or author doesn't change the URL.

Use case that brought me here: I wanted to email someone our list of research papers on child tax credits in the US. The above included one post from the UK.

**Comment by nikhilwoodruff** [comment]
Yes though just noting you can already do this manually: https://policyengine.org/us/research?locations=us&search=child+tax+credit

**Comment by MaxGhenis** [comment]
Thank you for filing this issue. Closing as part of the `policyengine-app` (v1) wind-down: policyengine.org is now served by [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2), and this repository is being archived, so this issue is frozen against a codebase that no longer deploys anywhere.

If this is still relevant on the current site, please re-file it against [`policyengine-app-v2`](https://github.com/PolicyEngine/policyengine-app-v2) — we'd welcome it there. Apologies that this sat open through the migration.
